src/components/molecules/RecipePreviewCard/RecipePreviewCard.tsx

```diff
@@ -1,26 +1,30 @@
 import Link from "next/link"
 
+import { FavoriteCount } from "../../atoms/FavoriteCount/FavoriteCount"
 import { RecipePreviewPicture } from "../../atoms/RecipePreviewPicture/RecipePreviewPicture"
 import { RecipePreviewText } from "../../atoms/RecipePreviewText/RecipePreviewText"
 
-interface RecipePreviewCardProps {
+type RecipePreviewCardProps = {
   recipe: {
     index: number
     url: string
```

Comment:
```suggestion
    imagePass: string
    linkUrl: string
```
LinkタグにセットするhrefとImageタグに渡すsrcの二つがあるので、

imagePass
linkUrl
みたいに識別できる命名でお願いいたしますー！

Reply:
ありがとうございます！
上記のようにProps修正しました